tests/add-delete-add.spec.ts

```diff
@@ -46,7 +44,5 @@ test("testing adding interventions, then deleting one, then adding another", asy
   });
   await page.getByRole("button", { name: "Finish" }).click();
 
-  await expect(
-    page.getByRole("button", { name: "1) Untitled point" }).isVisible()
-  );
+  await page.getByRole("button", { name: "1) Untitled route" }).isVisible();
```

Comment:
I was getting `Error: expect: Property 'then' not found.` We don't need this extra `expect`.